### PR TITLE
Add Philipino Peso to currencies

### PR DESCRIPTION
**Pull request by zenweasel**
Another life-changing PR from me

### CODE REVIEW

**Comment by aaronjudd** [comment]
@zenweasel since you just did this and it's fresh, would you mind adding an example to the docs for #538

**Comment by zenweasel** [comment]
D'oh. Yes. I should have done that.

File: packages/reaction-sample-data/private/data/Shops.json [hunk 1]
```diff
@@ -320,6 +320,10 @@
       "format": "%s%v",
       "symbol": "$"
     },
+    "PHP": {
+      "format": "%s%v",
+      "symbol": "PHP "
```
Comment (mikemurray) [inline packages/reaction-sample-data/private/data/Shops.json]: Add necessary spacing in `format`.

Comment (zenweasel) [inline packages/reaction-sample-data/private/data/Shops.json]: I tried that and it seemed to throw off alignment. That's why there is a space after symbol. But maybe I am missing something.

**Pull request merged**